mito-ai/mito_ai/completions/prompt_builders/chat_system_message.py

```diff
@@ -83,6 +85,62 @@ def create_chat_system_message_prompt() -> str:
 ==== 
 {CHAT_CODE_FORMATTING_RULES}
 
+
+<Example 1>
```

Comment:
We already have <Example 1> and <Example 2> in this file. It looks like the Example 1 that you added here should be kept and changed into Example 3, and the Example 2 that you copied here should be deleted because its duplicated with the original Example 2